guides/common/modules/proc_generating-a-list-of-installed-packages.adoc

```diff
@@ -0,0 +1,16 @@
+[id="Generating_a_List_of_Installed_Packages_{context}"]
+= Generating a List of Installed Packages
+
+Use this procedure to generate a list of installed packages in *Report Templates*.
+
+.Procedure
+
+. In the {ProjectWebUI}, navigate to *Monitor* > *Report Templates*.
+. Select the *Host - All Installed Packages* report template.
```

Comment:
By removing two steps, I also meant this line. Once you select the report template, it takes you to the report template page, but rest of the procedure talks about generating the list without actually being on the **Host - All Installed Packages** template page.

Reply:
Thanks @adamlazik1. I was wondering if you meant this one too. I figured I knock out the other lines and if this one needed to go as well, you would tell me. :)